src/Feature/Editor/Draw.re

```diff
@@ -86,7 +88,8 @@ let drawShapedText = {
     let font =
       Service_Font.resolveWithFallback(
         ~italic,
-        bold ? Revery.Font.Weight.Bold : Revery.Font.Weight.Normal,
+        // XXX: Should I make the bold one step upper than the current weight ?
+        bold ? Revery.Font.Weight.Bold : context.fontWeight,
```

Comment:
Good question, thank you for calling this out.

It does seem like a step-up would be ideal here - this case is when the syntax highlighting specifies that a token should be bold. What might be useful is to have a helper function that, given a current `Revery.Font.Weight`, steps it up one level - we could use that here. 

Something like:
```
let bolder: Revery.Font.Weight.t => Revery.Font.Weight.t = Revery.Font.Weight.(fun
| Thin => UltraLight
...
| Bold => ExtraBold
| ExtraBold => UltraBold
| UltraBold => Heavy
...
);
```

Reply:
Let's do this! 👌